installer/templates/phx_umbrella/apps/app_name_web/gitignore

```diff
@@ -1,14 +1,35 @@
-# App artifacts
-/_build
-/db
-/deps
-/*.ez
+# The directory Mix will write compiled artifacts to.
+/_build/
 
-# Generate on crash by the VM
+# If you run "mix test --cover", coverage assets end up here.
+/cover/
+
+# The directory Mix downloads your dependencies sources to.
+/deps/
+
+# Where 3rd-party dependencies like ExDoc output generated docs.
+/doc/
+
+# Ignore .fetch files in case you like to edit your project deps locally.
+/.fetch
+
+# If the VM crashes, it generates a dump, let's ignore it too.
 erl_crash.dump
 
-# Static artifacts
-/node_modules
+# Also ignore archive artifacts (built via "mix archive.build").
+*.ez
+
+# Ignore package tarball (built via "mix hex.build").
+<%= web_app_name %>-*.tar
+<%= if brunch do %>
+# If NPM crashes, it generates a log, let's ignore it too.
+npm-debug.log
+
+# The directory NPM downloads your dependencies sources to.
+/node_modules/
```

Comment:
This was on the original template, but it seems it should be `/assets/node_modules/`

Reply:
Yeah.

Reply:
Done in https://github.com/phoenixframework/phoenix/pull/2772/commits/61f9544e8322bc03c809678d0dbfe0b3a5a62e39